.github/workflows/continuous-deployment.yml

```
@@ -9,381 +9,118 @@ permissions:
+  build-staging:
+    name: "Build Staging Infra"
+    needs: "test-repo"
+    uses: ./.github/workflows/infra-build.yml
     with:
-      opentofu-version: ${{ vars.OPENTOFU_VERSION }}
-      infrastructure-directory: "./infrastructure"
       environment: "staging"
-      project-name: "rundog"
       backend-resource-group-name: ${{ vars.BACKEND_RG_STAGING }}
       backend-storage-account-name: ${{ vars.BACKEND_STORAGE_ACCOUNT_STAGING }}
+      debug-output: ${{ vars.PIPELINE_DEBUG_MODE == "true" }}
```

actionlint (1.7.7)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)